apps/messaging-api/routes/messages/index.ts

```diff
@@ -203,16 +220,51 @@ export default async function messages(app: FastifyInstance) {
         throw new NotFoundError(errorKey, "no template contents found");
       }
 
+      const allUsersLookup = allUsers.data.reduce<{
+        [userId: string]: (typeof allUsers.data)[0];
+      }>((acc, user) => {
+        acc[user.id] = user;
+        return acc;
+      }, {});
+
       // Create messages
-      let messageAndUserIds: Awaited<
+      let createdTemplateMessages: Awaited<
         ReturnType<typeof messageService.createTemplateMessages>
       > = [];
       try {
-        messageAndUserIds = await messageService.createTemplateMessages(
+        createdTemplateMessages = await messageService.createTemplateMessages(
           contents,
           allUsers.data.map((u) => ({ ...u, userId: u.id })),
           req.body.transportations,
           req.body.security,
+          req.body.scheduledAt,
+        );
+
+        await eventLogger.log(
+          MessagingEventType.createRawMessage,
+          createdTemplateMessages.map((msg) => {
+            const user = allUsersLookup[msg.userId];
+            return {
+              excerpt: msg.excerpt,
+              lang: msg.lang,
+              messageId: msg.messageId,
+              messageName: "", // message name isn't feature defined at this point
+              plainText: msg.plainText,
+              receiverFullName: `${user.firstName} ${user.lastName}`,
+              receiverPPSN: user.ppsn,
```

Comment:
```suggestion
              receiverPPSN: user.ppsn || "",
```

Just to use the same code as the `senderPPSN`
https://github.com/ogcio/life-events/pull/653/files#diff-c096eb68e16cee57687cdc4bd0b593f52454f8c34533dd16423f7b316c51708aR264